Makefile

```diff
@@ -414,6 +414,7 @@ build_web_js_api_parallel: install_rs_check_toolchain install_wasm_pack
 		wasm-pack build --release --target=web \
 		-- --features=boolean-client-js-wasm-api,shortint-client-js-wasm-api,integer-client-js-wasm-api,parallel-wasm-api,zk-pok \
 		-Z build-std=panic_abort,std
+	jq '.files += ["snippets"]' tfhe/pkg/package.json > tmp_pkg.json && mv -f tmp_pkg.json tfhe/pkg/package.json
```

Comment:
alignment isi not good, I think it breaks Makefiles

Reply:
It's ok, since it's not on the same line of command than the previous one.